lasagne/nonlinearities.py

```diff
@@ -46,12 +46,10 @@ def softmax(x):
 # tanh
 def tanh(x):
     """Tanh activation function :math:`\\varphi(x) = \\tanh(x)`
-
```

Comment:
Don't worry, that commit will be removed, it's part of a different PR (in which it also hasn't been addressed yet: https://github.com/Lasagne/Lasagne/pull/409#discussion_r39526174, but I guess I'll take care of it myself).